netstandard/ref/System.Net.cs

```diff
@@ -932,6 +964,12 @@ public partial class UploadValuesCompletedEventArgs : System.ComponentModel.Asyn
     public partial class WebClient : System.ComponentModel.Component
     {
         public WebClient() { }
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        [System.ObsoleteAttribute("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.", true)]
+        public bool AllowReadStreamBuffering { get { throw null; } set { } }
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        [System.ObsoleteAttribute("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.", true)]
+        public bool AllowWriteStreamBuffering { get { throw null; } set { } }
```

Comment:
Why do we want to add infrastructure API to the standard ?

Reply:
I don't think we do unless there are tools that need to call them.

Reply:
@davidsh any thoughts on this? For now, I've excluded all APIs that are marked as `[Obsolete("...", error: true]`

Reply:
The `WebClient` class is considered the oldest legacy HTTP class we have.  So, I'm fine with not including those Obsoleted APIs for this class.